src/runtime-tools/linux/setup.sh

```diff
@@ -6,7 +6,7 @@
 find .
 set -x
 
-INSTANCE_SETUP="/onefuzz/instance-specific-setup/setup.sh"
+INSTANCE_SETUP="/onefuzz/instance-specific-setup/linux/setup.sh"
```

Comment:
do we need this for the windows script too ?

Reply:
Not sure.  I'll check that tomorrow and open another PR if need be